### ISSUE TITLE
Bug in `partial()` when setting positional arguments

### ISSUE THREAD

**Issue by jonathn** [smarie/python-makefun#62]
When passing `preset_pos_args` (as opposed to `preset_kwargs`) to `partial()`..
the helper function `gen_partial_sig()` appears to wipe out the entire signature..

I believe the mistake is the use of `break` rather than `continue` here:
https://github.com/smarie/python-makefun/blob/028f5c9c93d4a231042a3001b2b54d5aa489d201/makefun/main.py#L1030

**Comment by smarie** [comment]
Thanks for reporting @jonathn  ! I'll have a look now

**Comment by smarie** [comment]
I was able to reproduce this issue with the following code:

```python
def f(a, b):
    return a+b

fp = partial(f, 0)
assert fp(-1) == -1
```

raises

`TypeError: f() takes 0 positional arguments but 1 was given`

**Comment by smarie** [comment]
This should be fixed in 1.11.0. Let me know if this is not the case ! I also took this opportunity to add easier editing parameters to `@wraps`. Thanks @jonathn for reporting!
